pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/NamespacesImpl.java

```diff
@@ -2452,6 +2452,30 @@ public void setMaxUnackedMessagesPerSubscription(String namespace, int maxUnacke
         return asyncPostRequest(path, Entity.entity(maxUnackedMessagesPerSubscription, MediaType.APPLICATION_JSON));
     }
 
+    @Override
+    public void removeMaxUnackedMessagesPerSubscription(String namespace)
```

Comment:
Do we need to also add a CLI command for this method?

Reply:
Ok i will add